sos/plugins/insights.py

```diff
@@ -26,7 +26,7 @@ class RedHatAccessInsights(Plugin, RedHatPlugin):
     def setup(self):
         log_size = self.get_option('log_size')
         self.add_copy_spec(self.conf_file)
-        self.add_copy_spec_limit('/var/log/redhat-access-insights/*.log',
+        self.add_copy_spec_limit('/var/log/redhat-access-insights/*.log*',
```

Comment:
This should really be under an `--all-logs` block, with a corresponding (default) branch that collects only the current (non-rotated) log file.

See e.g. the `dnf` plugin for an example that handles `--all-logs` and `--log-size`.